docs/examples/mtls/README.md

```diff
@@ -0,0 +1,43 @@
+# mTLS Example
+
+You can enable mTLS by providing the necessary TLS certificates and keys in Secret objects.
+You must set `.spec.tls.secretName` to the name of a secret containing the RabbitMQ server's TLS certificate and key,
+and set `spec.tls.caSecretName` to the name of a secret containing the certificate of the Certificate Authority which
+has signed the certificates of your RabbitMQ clients.
+
+First, you need to create the server Secret like this (assuming you already have `server.pem` and `server-key.pem` files):
```

Comment:
The command below gives an indication of what we mean by server.pem and server-key.pem. But I think it's better to mention what we mean by them in the assumption.